modules/apps/portal-security-sso/portal-security-sso-openid-connect-impl/bnd.bnd

```diff
@@ -1,6 +1,10 @@
 Bundle-Name: Liferay Portal Security SSO OpenID Connect Implementation
 Bundle-SymbolicName: com.liferay.portal.security.sso.openid.connect.impl
+<<<<<<< Updated upstream
```

Comment:
Hey @alvarosaugarlr , this is a merge conflict.

Reply:
Sorry! I don't know what I have made wrong, because in my laptop is merged

Reply:
New commit to solved.

Reply:
@arthurchan35